integration-cli/docker_cli_swarm_test.go

```diff
@@ -68,14 +77,22 @@ func (s *DockerSwarmSuite) TestSwarmInit(c *check.C) {
 		return sw.Spec
 	}
 
+	tempFile, err := ioutil.TempFile("", "testfile")
+	c.Assert(err, checker.IsNil)
+	defer os.Remove(tempFile.Name())
+	tempFile.Write([]byte("fakecert"))
+
 	cli.Docker(cli.Args("swarm", "init", "--cert-expiry", "30h", "--dispatcher-heartbeat", "11s",
-		"--external-ca", "protocol=cfssl,url=https://something.org"),
+		"--external-ca", "protocol=cfssl,url=https://something.org",
+		"--external-ca", fmt.Sprintf("protocol=cfssl,url=https://somethingelse.org,cacert=%s", tempFile.Name())),
 		cli.Daemon(d.Daemon)).Assert(c, icmd.Success)
 
 	spec := getSpec()
 	c.Assert(spec.CAConfig.NodeCertExpiry, checker.Equals, 30*time.Hour)
 	c.Assert(spec.Dispatcher.HeartbeatPeriod, checker.Equals, 11*time.Second)
-	c.Assert(spec.CAConfig.ExternalCAs, checker.HasLen, 1)
+	c.Assert(spec.CAConfig.ExternalCAs, checker.HasLen, 2)
+	c.Assert(spec.CAConfig.ExternalCAs[0].CACert, checker.IsNil)
+	c.Assert(reflect.DeepEqual(spec.CAConfig.ExternalCAs[1].CACert, []byte("fakecert")), checker.True)
 
 	c.Assert(d.Leave(true), checker.IsNil)
 	time.Sleep(500 * time.Millisecond) // https://github.com/docker/swarmkit/issues/1421
```

Comment:
It should be safe to remove this now, since the underlying swarmkit issue (actually a etcd/raft issue) has been fixed. We can do it in a different PR though if you prefer that. Let me know if I should open one.